<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>project</class>
 <widget class="QDialog" name="project">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>304</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>思源黑体 CN Regular</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>新建项目</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>src/images/tree.png</normaloff>src/images/tree.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="line_url">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="line_name">
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="btn_file">
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>打开文件夹</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>项目名称</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>项目路径</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="btn_box">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>项目文件</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="line_file">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>思源黑体</family>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_box</sender>
   <signal>accepted()</signal>
   <receiver>project</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_box</sender>
   <signal>rejected()</signal>
   <receiver>project</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line_file</sender>
   <signal>textChanged(QString)</signal>
   <receiver>project</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
